superset/assets/cypress/integration/dashboard/url_params.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { WORLD_HEALTH_DASHBOARD } from './dashboard.helper';
+
+export default () => describe('dashboard url params', () => {
+  let sliceIds = [];
+
+  beforeEach(() => {
+    cy.server();
+    cy.login();
+
+    cy.visit(WORLD_HEALTH_DASHBOARD + '?param1=123&param2=abc');
```

Comment:
Here I would have used `qs` introduced in cypress `3.5.0` (https://docs.cypress.io/guides/references/changelog.html#3-5-0), but since we're still on `3.4.1`, I had to put it in the URL by hand. If there are no objections, I can bump to `3.6.1` which would make this test slightly cleaner (being able to assert against the same object that's being passed to visit).

Reply:
👍